eng/Publishing.props

```diff
@@ -3,4 +3,32 @@
    <PropertyGroup>
       <PublishingVersion>3</PublishingVersion>
    </PropertyGroup>
+
+  <PropertyGroup>
+    <PublishDependsOnTargets>_ResolvePublishRoslynNuGetPackages;$(PublishDependsOnTargets)</PublishDependsOnTargets>
+  </PropertyGroup>
+
+  <Target Name="_ResolvePublishRoslynNuGetPackages"
+          DependsOnTargets="BeforePublish"
+          BeforeTargets="PublishToAzureDevOpsArtifacts">
+    <!--
+      During PR Validation we only need to publish symbols with Arcade,
+      since our packages are published separately to the CoreXT feed.
+    -->
+    <ItemGroup Condition=" '$(PreReleaseVersionLabel)' == 'pr-validation' ">
```

Comment:
should the condition be on the overall target?

Reply:
Happy to move it up.